### PR TITLE
Add bug, feature, and templates as well as link to planet community for questions

### LINKED ISSUE #730: Add PR and bug report templates
To streamline the PR and PR review process, add a PR template. example: [stac](https://github.com/radiantearth/stac-spec/blob/master/.github/pull_request_template.md)

And to ensure adequate information for bug reports, add a bug report template. example [pryproj](https://github.com/pyproj4/pyproj/blob/main/.github/ISSUE_TEMPLATE/bug_report.md)

### PR DESCRIPTION
**Pull request by jreiberkyle**
1. Instead of being met with a blank issue when creating issues, contributors will be met with the options: Bug Report, Feature Request, and Ask A Question. Bug Report and Feature Request will be forms for creating issues with labels bug/proposal, respectively. Ask A Question is a link to planet community.
2. When a PR is initiated, instead of being met with a blank page, contributors will be met with a form to be filled out and checklist to confirm.

Closes #730

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Ask a question
+    url: https://community.planet.com/planet-s-community-forum-3
```
Comment (cholmes) [inline .github/ISSUE_TEMPLATE/config.yml:3]: We probably should turn off our 'discussions' in github if we're pointing people at Planet community (which I do think is probably the right thing).

**Review by cholmes — APPROVED** [review]
All looks good! One minor incidental comment, that's outside the scope of this PR.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (jreiberkyle) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Agreed. Is pointing to planet community the right thing to do? Will we have someone from this team monitoring the forum or will a monitor connect with someone from our team if an SDK question comes up?

Comment (cholmes) [reply .github/ISSUE_TEMPLATE/config.yml:3]: Yeah, I think we should be able to monitor, or at least be notified if questions about the SDK come in.

**Pull request merged**